tests/config/test_data.config

```diff
@@ -677,6 +677,10 @@ params {
                 idxml1                          = "${params.test_data_base}/data/proteomics/openms/HepG2_rep1_small.idXML"
                 idxml2                          = "${params.test_data_base}/data/proteomics/openms/HepG2_rep2_small.idXML"
             }
+            'pdb' {
+                tim1_pdb                        = "${params.test_data_base}/data/proteomics/pdb/1tim.pdb"
+                tim8_pdb                        = "${params.test_data_base}/data/proteomics/pdb/8tim.pdb"
```

Comment:
Variable names cannot start with integer numbers (breaks all kinds of stuff) so instead of 1tim_pdb and 8tim_pdb I renamed these to tim1_pdb and tim8_pdb respectively